SPARCore/GADArticle.m

```diff
@@ -2,6 +2,48 @@
 
 @implementation GADArticle
 
++ (void) fetchArticlesWithPublication: (NSString *)publicationId
+                                        completionHandler:(void(^_Nonnull)(NSArray<GADRemoteModel *>
+                                                                           *_Nullable models,
+                                                            NSError *_Nullable error))completion {
+    //Should we create NSDictionary parameters within each method and just request the publicationId
+    //from the front end?
+
+}
+
+- (void) populateArticleWithId: (NSString *)articleId
```

Comment:
Yes, it looks like you'll have an `articleId` property, even in the `ShortArticleModel`s. Thus, you don't need that as a parameter.